Server/ReasnAPI/ReasnAPI/Services/ImageService.cs

```diff
@@ -130,7 +130,7 @@ public void DeleteImageById(int id)
 
     public void DeleteImageRelatedToEvent(int id, string slug)
     {
-        var @event = context.Events.FirstOrDefault(r => r.Id == id && r.Slug == slug);
+        var @event = context.Events.FirstOrDefault(r => r.Slug == slug);

```

Comment:
Should we really use reserved keywords with @ sign? Maybe change it to relatedEvent or so

Reply:
done